src/App.tsx

```diff
@@ -1,61 +1,136 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './components/TSTypes/Todo';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function findByIdHelper(userId: number) {
+  return usersFromServer.find(({ id }) => id === userId);
+}
+
+const todosWithUsers = todosFromServer.map(todo => {
+  const user = findByIdHelper(todo.userId);
+
+  return {
+    ...todo,
+    user,
+  };
+});
+
+export const App: React.FC = () => {
+  const [toDoList, setToDoList] = useState<Todo[]>(todosWithUsers);
+
+  const [title, setTitle] = useState<string>('');
+  const [userId, setUserId] = useState<number>(0);
+
+  const [titleError, setTitleError] = useState<boolean>(false);
+  const [userError, setUserError] = useState<boolean>(false);
```

Comment:
Boolean variables should have a prefix `is` or `has`. Also, you don't need to set the type manually, as you pass a boolean as a default value typescript know that it will have this type